service/controller/v1/prometheus/testdata/scrapeconfig.golden

```diff
@@ -1,4 +1,5 @@
 - job_name: guest-cluster-xa5ly-apiserver
+  honor_timestamps: false
```

Comment:
The changes in this here golden file are the only test changes, and amount to adding and removing a few defaults. Easy.